app/src/main/java/zapsolutions/zap/util/MonetaryUtil.java

```diff
@@ -360,6 +360,32 @@ public String convertPrimaryToSatoshi(String primaryValue) {
         }
     }
 
+    /**
+     * Converts the supplied value to satoshis. The exchange rate of the secondary currency is used.
+     *
+     * @param secondaryValue
+     * @return String without grouping or fractions
+     */
+    public String convertSecondaryToSatoshi(String secondaryValue) {
```

Comment:
I guess we could simplify this by merging `convertPrimaryToSatoshi` and `convertSecondaryToSatoshi` as it is basically the same code with just one boolean difference. 
something like e.g. 
`convertToSatoshi(amout, usePrimary)`

Reply:
That's true. We actually should refactor the whole util and add unit tests when we have time. Conversions are to dangerous not to be tested.